src/AsyncGenerator.yml

```diff
@@ -109,6 +109,12 @@
     typeConversion:
     - conversion: Ignore
       name: EnumerableImpl
+    - conversion: Ignore
+      name: SystemTransactionContext
+      containingTypeName: AdoNetWithSystemTransactionFactory
```

Comment:
> `ManualResetEvent.Wait` call which has an async counterpart but is currently not detected by the async generator

There is no `WaitAsync` method for the [ManualResetEvent](https://msdn.microsoft.com/en-us/library/system.threading.manualresetevent(v=vs.110).aspx) or I am missing something?

Reply:
My bad, previously it was a `SemaphoreSlim` instead of a `ManualResetEventSlim`. The semaphore has async methods, but indeed the reset does not, it just have some overloads taking cancellation token.